controller/backing_image_manager_controller.go

```diff
@@ -61,6 +61,9 @@ type BackingImageManagerController struct {
 	backoffMap sync.Map
 
 	versionUpdater func(*longhorn.BackingImageManager) error
+
+	replenishLock             *sync.Mutex
+	inProgressReplenishingMap map[string]struct{}
```

Comment:
Does the controller need to do cleanup when somehow `bim.deletionTimestamp != nil`?

Reply:
This logic is copied from replica
https://github.com/longhorn/longhorn-manager/blob/master/controller/replica_controller.go#L59

We don't cleanup the `inProgressRebuildingMap ` there
what would be the possible consequence if we don't clean it up?

Reply:
fixed.